linux_os/guide/system/software/sudo/sudo_custom_logfile/tests/logfile_enabled.pass.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# platform = multi_platform_all
+
+echo "Defaults logfile=/var/log/sudo.log" >> /etc/sudoers
```

Comment:
Should we also consider cases when multiple options are defined in a single `Defaults`?
The current `option_regex_suffix` will not in these cases. I think it will match the following options as part of the path.
E.g: `Defaults logfile=/var/log/sudo.log,noexec`

Reply:
Do our current `sudo` rules support that? My understanding is no, because we match on `Defaults <option>` -- so if you have an entry like `Defaults logfile=/var/log/sudo.log,noexec` -- it'll only match for `logfile` and any rule for `noexec` will fail because `Defaults noexec` is missing from the sudo rules?

Regarding compliance... If there isn't a `noexec` rule and it was added by the admin, I'm not sure how I feel about that. CIS doesn't appear to mention that scenario and you do get into dangerous territory -- what if your value is `Defaults logfile=/var/log/noexec/sudo.log,something`? Would you match on `Defaults noexec` and `Defaults.*,noexec`? Is there a way to escape a comma in a parameter value? &c. I'd prefer to stick with simple matches, IMO. I dunno. 

My 2c. but if we do want to support it, I think it is outside the scope of this particular PR. Thoughts?

Reply:
> Do our current `sudo` rules support that? My understanding is no, because we match on `Defaults <option>` -- so if you have an entry like `Defaults logfile=/var/log/sudo.log,noexec` -- it'll only match for `logfile` and any rule for `noexec` will fail because `Defaults noexec` is missing from the sudo rules?

They do work with multiple options. When `OPTION_REGEX` is `\w+`, it doesn't match the commas.
And the OVAL accounts for boundaries too, which should help ensure it is not taking the whole line:
https://github.com/ComplianceAsCode/content/blob/master/shared/templates/sudo_defaults_option/oval.template#L16

This rule has tests with multiple values per line:
https://github.com/ComplianceAsCode/content/tree/master/linux_os/guide/system/software/sudo/sudo_add_umask/tests

> My 2c. but if we do want to support it, I think it is outside the scope of this particular PR. Thoughts?

Well, it could be outside of scope of this PR.

Reply:
Ah, d'oh, I missed the `.*` :D 

I agree, I'll make changes :-)